src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -204,7 +204,9 @@ private void validateSchema(final EventType eventType) throws InvalidEventTypeEx
         try {
             final JSONObject schemaAsJson = new JSONObject(eventType.getSchema().getSchema());
 
-            if (hasReservedField(eventType, schemaAsJson, "metadata")) {
+            if (eventType.getCategory() == EventCategory.BUSINESS
```

Comment:
As some food for thought, I would say that Business event is more confusing than Data event because it mixes user data and Nakadi data under the same "level". For example, a business event has "metadata" and all other properties in the same Json properties. But Data event "envelops" all user data inside "data" field. It seems to be a good idea to make it mandatory for all Events to have a "common" envelope. It would make everything more homogeneous. Event "undefined" should be enveloped, in my opinion, so that we could use all partition strategies with it.